atramhasis/templates/tree.jinja2

```diff
@@ -212,7 +212,7 @@
         d3.event.preventDefault();
         if (d.concept_id) {
           var basepath = '{{ request.route_path('concept', scheme_id= scheme_id, c_id = '') }}';
-          window.location.href = basepath + d.concept_id;
+          window.location.href = basepath + d.concept_id +'?&view_tree';
```

Comment:
die '&' hoeft hier eigenlijk niet, dat is voor als er meerdere query params zijn